client/src/app/components/employees/employee-list/employee-list.component.html

```diff
@@ -62,6 +62,13 @@
       <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
     </table>
 
-    <mat-paginator [pageSizeOptions]="[10, 25, 50, 100]"></mat-paginator>
+    <mat-paginator
+    [length]="length"
+    [pageSize]="pageSize"
+    [pageSizeOptions]="[25,50,100]"
```

Comment:
Why did we remove 10 as an option for the size?

Reply:
The smallest page size is currently set to 25. It is also to stay consistent with the other tables

Reply:
Perfect thanks!